File: step6/1065.c
#include <stdio.h>

int ishansu(int X)
{
    if (X / 100) {
        if (X / 100 + X % 10 == 2 * (X % 100 / 10))
            return 1;
        else
            return 0;
    } else {
        return 1;
    }
}
int main(void)
{
    int N, cnt = 0;
    scanf("%d", &N);
    for (int x = 1; x < N + 1; x++) {
        cnt = cnt + ishansu(x);
    }
    printf("%d", cnt);
    return 0;
}

File: step6/1065.py
def ishansu(X):
    if X // 100:
        return 1 if X // 100 + X % 10 == 2 * (X % 100 // 10) else 0
    else:
        return 1


N = int(input())
cnt = 0
for x in range(1, N + 1):
    cnt += ishansu(x)
print(cnt)